### ISSUE TITLE
Popover becomes sticky

### ISSUE THREAD

**Issue by RAIbrahim360** [Akryum/floating-vue#1003]
https://github.com/Akryum/floating-vue/assets/53712362/db83e145-02b9-42c2-bcbf-4a26417fd7f3

"floating-vue": "^1.0.0-beta.19". I'm using Dropdown component. The bug appears when i'm using placement=right start or left start.

**Comment by RAIbrahim360** [comment]
@Akryum Any progress on that?

**Comment by Akryum** [comment]
You need to set `prevent-overflow` to `false`

**Comment by RAIbrahim360** [comment]
@Akryum No! It isn't solution because now popover goes outside of boundary but is should reposition. It's bug because when i set placement as top or bottom it works but bug appears when placement right or left. When this library used popper.js there's no bug.

**Comment by Akryum** [comment]
Then try setting `shift` to `false`
https://floating-vue.starpad.dev/api/#shift

**Comment by mrleblanc101** [comment]
I've got the same question, anybody found a solution ?

`<VDropdown :distance="4" :container="false">`
<img width="261" height="265" alt="Image" src="https://github.com/user-attachments/assets/8e42f94b-73c8-4a81-8be2-e16b9e60ddc2" />

`<VDropdown :shift="false" :distance="4" :container="false">`
<img width="261" height="265" alt="Image" src="https://github.com/user-attachments/assets/4b00f80d-be5e-4fda-ae57-ef11d6761bc0" />

Also, I had to add:
```
.v-popper--theme-dropdown {
    @apply relative;
}
```
Otherwise the popper wouldn't be clipped by the `overflow: auto;` 🤔 

<img width="424" height="293" alt="Image" src="https://github.com/user-attachments/assets/ebee335c-c3f6-4284-873d-00ea2e9676b0" />
